components/index/LoginForm.tsx

```diff
@@ -75,7 +75,7 @@ const LoginForm = () => {
         ) : null}
       </FormGroup>
       <FormBtnContainer>
-        <Button $purple={true} onClick={formik.handleSubmit} type="submit">
+        <Button purple={true} onClick={formik.handleSubmit} type="submit">
```

Comment:
`nit`

```suggestion
        <Button purple onClick={formik.handleSubmit} type="submit">
```

I think that works too. No need to specify `purple={true}`. Correct me if wrong

Reply:
![image](https://user-images.githubusercontent.com/91857581/188920048-c87fb609-1b4d-45d8-a55e-cabf06567d4c.png)
yep, your suggestion works the same
although I'm getting some kind of error now looking like this 
![image](https://user-images.githubusercontent.com/91857581/188920352-2f7958ef-2370-46cf-b9e5-b0efbfb29eac.png)
and I have no idea what that is about yet